backend/src/main/scala/com/softwaremill/bootzooka/passwordreset/PasswordResetService.scala

```diff
@@ -27,7 +27,7 @@ class PasswordResetService(
 
   def forgotPassword(loginOrEmail: String): ConnectionIO[Unit] = {
     userModel.findByLoginOrEmail(loginOrEmail.lowerCased).flatMap {
-      case None => Fail.NotFound("user").raiseError[ConnectionIO, Unit]
+      case None => ().pure[ConnectionIO]
```

Comment:
maybe add a debug-log so that it's visible in the logs that the user hasn't been found? (as it's not visible in the UI)